CNAME

```diff
@@ -1 +0,0 @@
-fossasia.org
```

Comment:
Why?

Reply:
Revert it and provide preview URL with gh pages.

Reply:
Also, @Shizerq Try and merge your commits.